universal-login-relayer/test/integration/sql/EmailConfirmationsStore.test.ts

```diff
@@ -0,0 +1,51 @@
+import {getKnexConfig} from '../../testhelpers/knex';
+import {EmailConfirmationsStore} from '../../../src/integration/sql/services/EmailConfirmationsStore';
+import {EmailConfirmation, sleep} from '@unilogin/commons';
+import {expect} from 'chai';
+
+describe('UNIT: EmailConfirmationsStore', () => {
+  const knex = getKnexConfig();
+  const emailConfirmationsStore = new EmailConfirmationsStore(knex);
+
+  it('Should add emailConfirmation to database and get it from it', async () => {
+    const storedEmail = 'newEmail@email.com';
+    const storedEmailConfirmation = {
+      email: storedEmail,
+      ensName: 'bob.unilogin.eth',
+      code: '1234',
+      isConfirmed: false,
+      created_at: new Date(),
+    } as EmailConfirmation;
+
+    const [email] = await emailConfirmationsStore.add(storedEmailConfirmation);
+    expect(email).be.deep.eq(storedEmail);
+    expect(await emailConfirmationsStore.get(email)).be.deep.eq(storedEmailConfirmation);
+  });
+
+  it('Should add two emailConfirmations to database and get the second one', async () => {
+    const storedEmail = 'newEmail@email.com';
+    const storedEmailConfirmation = {
+      email: storedEmail,
+      ensName: 'bob.unilogin.eth',
+      code: '1234',
+      isConfirmed: false,
+      created_at: new Date(),
+    } as EmailConfirmation;
+
+    let [email] = await emailConfirmationsStore.add(storedEmailConfirmation);
+    expect(email).be.deep.eq(storedEmail);
+
+    await sleep(1);
```

Comment:
why is this sleep for?

Reply:
We want to have different created_at dates and sometimes they were the same

Reply:
Done